packages/react/src/color-mode/InvertedMode.js

```diff
@@ -0,0 +1,23 @@
+import React, { forwardRef } from 'react';
+import { Box } from '../box';
+import ColorModeProvider from './ColorModeProvider';
+import useColorMode from './useColorMode';
+
+const InvertedMode = forwardRef((props, ref) => {
+  const [colorMode] = useColorMode();
+  const invertedColorMode = colorMode === 'light' ? 'dark' : 'light';
+
+  return (
+    <ColorModeProvider value={invertedColorMode}>
```

Comment:
This is pretty minor. I only just noticed this PR while I was reading up on some of the newest components. Could we have made use of existing DarkMode/LightMode components here?


```
import LightMode from './LightMode';
import DarkMode from './DarkMode';

const InvertedMode = forwardRef((props, ref) => {
  const [colorMode] = useColorMode();
  const InvertedColorMode = colorMode === 'light' ? DarkMode : LightMode;
  return <InvertedColorMode ref={ref} {...props} />;
};
```